### DISCUSSION TITLE
My trained VITS model size is 990mb while coque-tts pre-trained VITS file size is 148mb

### DISCUSSION THREAD

**Discussion by iamkhalidbashir** [coqui-ai/TTS#2413] [General Q&A]
How to reduce the file size of vits trained model size to one like available in coque-tts pre-trained repo.

I would assume we might have to remove some non-needed data from the model file that will not allow us to use it for continuing training but will reduce size.

Any help would be much appreciated! Thanks.

**Comment by erogol** [comment]
discriminator is not included in the release version.

**Reply by iamkhalidbashir** [reply]
How to remove the discriminator from my trained model too?

**Reply by iamkhalidbashir** [reply]
Also are there pre-trained models with discriminators available from coqui-tts mode zoo to speed up fine-tuning

**Reply by iamkhalidbashir** [reply]
> How to remove the discriminator from my trained model too?

I think I found answer to my first question
https://github.com/coqui-ai/TTS/discussions/1890#discussioncomment-3446831
